doc/lsst.ap.pipe/pipeline-bps.rst

```diff
@@ -0,0 +1,141 @@
+  # Further arguments are given in the calls to 'runQuantumCommand' below
+  payload:
+    # Specifies whether to run --init-only on each pipetask run before doing the real pipetask run
+    runInit: true
+    # BE SURE TO UPDATE THIS it will set the output collection name
+    payloadName: my_example_name
+    # -b
+    butlerConfig: /datasets/hsc/gen3repo/rc2w50_ssw02/butler.yaml
+    # -i
+    inCollection: HSC/calib,HSC/raw/all,refcats,u/diffim_sprint/templates_bestThirdSeeing
+    # -o  # Note: must specify outCollection with timestamp so you don't get a zillion sub-runs
+    output : 'u/${USER}/{payloadName}'
+    outCollection: '{output}/{timestamp}'
+    # -d
+    dataQuery: 'exposure IN (11690, 11692) AND detector in (49, 50)'  # mini run
+  
+  # Various things for bps to customize about each pipeline task
+  pipetask:
+    # Initialization (optional, but a good idea; treated like its own pipeline task)
+    pipetaskInit:
+      runQuantumCommand: '${CTRL_MPEXEC_DIR}/bin/pipetask --long-log run -b {butlerConfig} -i {inCollection} --output {output} --output-run {outCollection} --init-only --skip-existing --register-dataset-types --qgraph {qgraphFile} --clobber-partial-outputs --no-versions'
+    # Option to list other pipeline tasks being run here with default overrides
+    imageDifference:
+      requestMemory: 4096
+  
+  # Condor backend stuff
+  wmsServiceClass: lsst.ctrl.bps.wms.htcondor.htcondor_service.HTCondorService
+  clusterAlgorithm: lsst.ctrl.bps.quantum_clustering_funcs.single_quantum_clustering
+  
+  # Create quantum graph for use by all tasks
+  createQuantumGraph: '${CTRL_MPEXEC_DIR}/bin/pipetask qgraph -d "{dataQuery}" -b {butlerConfig} -i {inCollection} -p {pipelineYaml} -q {qgraphFile} --qgraph-dot {qgraphFile}.dot'
+  
+  # Main pipetask run call
+  runQuantumCommand: '${CTRL_MPEXEC_DIR}/bin/pipetask --long-log run -b {butlerConfig} -i {inCollection} --output {output} --output-run {outCollection} --extend-run --skip-init-writes --qgraph {qgraphFile} --clobber-partial-outputs --no-versions'
+
+Notes on the yaml file
+----------------------
+
+* The `computeSite` option determines where your jobs will run. As of now (April 2021) the typical choice will be `ncsapool`. Other options may be possible and/or make more sense for you, depending on what you're doing. It is best to ask the bps expoerts about that.
+* Note that you must use the long option names in a yaml file for the apparopriate pipetask options, e.g. `butlerConfig` instead of `-i`, `dataQuery` instead of `-d`, etc.
+* You can request default resource requirements such as memory or run time at the top level of the yaml (see the `requestMemory` line above), but you can give other values for specific task types if you want (see the higher requestMemory value in the imageDifference section under pipetask).
+* Don't forget to set your butler, input and output collections, and any other absolute paths according to your own work area.
```

Comment:
Good place for a note about "MODIFY THIS" (or whatever you end up calling it).

Reply:
I think I removed that MODIFY THIS bit.